Task 9/script.js

```diff
@@ -9,3 +9,21 @@ budget: number
 Metodas: 
 wasExpensive() - jeigu filmo "budget" yra daugiau nei 100 000 000 mln USD, tada grąžins true, kitu atveju false. 
 ------------------------------------------------------------------------------------------------------ */
+
+class Movie {
+    constructor(title, director, budget) {
+      (this.title = title),
+      (this.director = director),
+      (this.budget = budget),
+      (this.wasExpensive = function () {
+        if (this.budget > 100000000) {
+        return true;
+        } else {
+        return false;
+        }
+      });
+    }
+  }
+
+  const expensiveMovie = new Movie ("noName", "noDirector", 100000001);
+  console.log(expensiveMovie.wasExpensive(expensiveMovie));
```

Comment:
Teiginius skiriame kabliataškiais, ne kableliais, formatavimo įrankis matomai bandė ištaisyti šitą klaidą pridėdamas visur skliaustelius, bet atrodo labai negerai.

Struktūrą

```js
if (sąlyga) {
  return true;
} else {
  return false;
}
```

galima supaprastinti į

```js
return sąlyga;
```

ir analogiškai

```js
if (sąlyga) {
  return false;
} else {
  return true;
}
```

į

```js
return !sąlyga;
```